### PR TITLE
visualization website

### PR DESCRIPTION
**Pull request by malytomas**

### CODE REVIEW

File: web/register.htm [hunk 1]
```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
```
Comment (tivvit) [inline web/register.htm:1]: style: can you please autoformat this?

Comment (tivvit) [inline web/register.htm:1]: style: why not .html?

Comment (tivvit) [inline web/register.htm:1]: style: it would be nicer to move the js to a separate file

File: web/style.css [hunk 2]
```diff
@@ -0,0 +1,33 @@
+html, body {
```
Comment (tivvit) [inline web/style.css:1]: have you considered sass (or less)?

It is easy to use and I think you would love it

File: server/main.go [hunk 3]
```diff
@@ -43,28 +43,46 @@ func serve(ctx context.Context, wg *sync.WaitGroup) {
 	}
 }
 
+func addDefaultHeaders(fn http.HandlerFunc) http.HandlerFunc {
```
Comment (tivvit) [inline server/main.go:46]: nice you are editing go now!

File: web/register.htm (hunk 1 above)
Comment (malytomas) [reply web/register.htm:1]: just a personal preference ;)

File: web/style.css (hunk 2 above)
Comment (malytomas) [reply web/style.css:1]: I have had enough of "new exciting modern" technologies in the last week, and I see no use for _less_ in such a small scope project.

File: web/register.htm (hunk 1 above)
Comment (malytomas) [reply web/register.htm:1]: done

**Pull request merged**